perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_optional_login_namespace.java

```diff
@@ -0,0 +1,23 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleAbstract;
+
+/**
+ * Attribute, that contains login, only if it is available in the namespace.
+ *
+ * @author Matej Hakoš
+ */
+public class urn_perun_user_attribute_def_virt_optional_login_namespace extends UserVirtualAttributesModuleAbstract {
```

Comment:
We'll need the getValue() implementation that returns user's `def:login-namespace:XY` attribute value - otherwise the implementation from parent class `UserVirtualAttributesModuleAbstract` would be used and returned value would be always null (no real value is stored in this attribute, it'll always be computed on demand).

Could you please add tests for this module? :rocket: